### ISSUE TITLE
It is no longer possible to purchase CYW920819EVB-02 dev boards to run the BIAS PoC

### ISSUE THREAD

**Issue by jsmif** [francozappa/bias#21]
It would be great if you could port the PoC to an alternate device

**Comment by jsmif** [comment]
It looks like the [CYW920819M2EVB-01](https://www.infineon.com/cms/en/product/evaluation-boards/cyw920819m2evb-01/) may be the replacement for CYW920819EVB-02 ? Can you confirm that it would work if I purchased one?

**Comment by ghost** [comment]
@jsmif Could you let me know if you have tested this, and got it to work.

**Comment by jsmif** [comment]
@0xalim I don't have funding to purchase hardware speculatively if I don't know it will work. I was hoping that @francozappa  would purchase it and try it out.

**Comment by ghost** [comment]
@jsmif I have it, maybe we can work together to see if it would work. If you are interested and willing. If so, give me way we can easily communicate together on this.

**Comment by jsmif** [comment]
@0xalim If you have the board already, did you just try the existing instructions and they didn't work? If so, that feels like it should get a new issue filed.

**Comment by ghost** [comment]
The addresses for parameters found in AF.json are correct, but I do not know whether or not the addresses in the assembly code (bias-template.py) is correct or not.

I do not know how to validate whether or not it works besides checking LMP packets, and I am getting Authentication Failure in some cases. I can send a wireshark capture if you would like. Let me know an email or w/e to send to.

**Comment by sgxgsx** [comment]
@0xalim @jsmif Were you able to reproduce BIAS attack on another board?

**Comment by ntsimerekis** [comment]
@0xalim @jsmif I second, this. Please tell us if you've had success.

**Comment by hnznhr** [comment]
Is it possible to implement a BIAS attack on CYW920819M2EVB-01?

**Comment by LisaLav** [comment]
I want to say that I have successfully gotten the BIAS attack to work on a CYW920819M2EVB-01 board. It required some fiddling about with the code, so I plan on releasing how I got it to work, but it is definitely doable.

**Comment by francozappa** [comment]
Awesome @LisaLav, please ping me when you release your guide, as there are other people interested!

**Comment by LisaLav** [comment]
@francozappa Will do! It will be a bit before I release it as I want to do some more testing, but I will put a message here once it is done.

**Comment by csdragon83** [comment]
@LisaLav Hi! Do you still test your changes? I have the same problem! that I am not able to get a "-02 dev board" anymore.
Would be really great to get a working software for that new device!
